spec/javascripts/ActiveDocs/OAS3Autocomplete.spec.jsx

```diff
@@ -2,7 +2,7 @@
 
 // $FlowIgnore[cannot-resolve-module] OAS3Autocomplete module not found by Flow since not included in name-mapper.
 import { autocompleteOAS3 } from 'OAS3Autocomplete'
-import * as utils from 'utilities/utils'
+import * as utils from 'utilities/fetchData'
```

Comment:
why `import * as utils from 'utilities/fetchData'` here? 
```suggestion
import * as utils from 'utilities'
```

Reply:
It needs to be only the fetch module and I wanted the fewest line changes possible.